src/main/java/edu/wpi/grip/core/sources/WebCamSource.java

```diff
@@ -0,0 +1,131 @@
+package edu.wpi.grip.core.sources;
+
+
+import com.google.common.eventbus.EventBus;
+import edu.wpi.grip.core.OutputSocket;
+import edu.wpi.grip.core.SocketHint;
+import edu.wpi.grip.core.Source;
+import org.bytedeco.javacpp.opencv_core.Mat;
+import org.bytedeco.javacv.Frame;
+import org.bytedeco.javacv.FrameGrabber;
+import org.bytedeco.javacv.OpenCVFrameConverter;
+import org.bytedeco.javacv.OpenCVFrameGrabber;
+
+import java.util.Optional;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+/**
+ * Provides a way to generate a constantly updated {@link Mat} from a WebCam attached
+ * to the users computer.
+ */
+public class WebCamSource implements Source {
```

Comment:
I tried that, Intellij kept complaining about spelling. So I converted it to the other one.
You still want me to change it?